File: fixtures/scale/dart/large/lib/ciphers/affine/affine_key.dart
class AffineKey {
  final int a;
  final int b;

  AffineKey(this.a, this.b);

  int inverseOfA() {
    for (var candidate = 1; candidate < 26; candidate++) {
      if ((a * candidate) % 26 == 1) return candidate;
    }
    throw StateError('a is not coprime with 26');
  }

  factory AffineKey.defaultKey() => AffineKey(5, 8);
}

File: fixtures/scale/kotlin/large/src/cryptobench/ciphers/affine/AffineKey.kt
package cryptobench.ciphers.affine

data class AffineKey(val a: Int, val b: Int) {

    fun inverseOfA(): Int {
        for (candidate in 1 until 26) {
            if (a * candidate % 26 == 1) return candidate
        }
        error("a is not coprime with 26")
    }

    companion object {
        fun default(): AffineKey = AffineKey(5, 8)
    }
}
